standards-in-genomic-sciences.csl

```diff
@@ -182,7 +182,7 @@
           <text macro="editor" prefix=" " suffix="."/>
           <group prefix=" " suffix=".">
             <text variable="container-title" font-style="italic" form="short" suffix=" "/>
-            <group delimiter=";" prefix=" ">
+            <group delimiter=";">
               <date variable="issued">
                 <date-part name="year"/>
               </date>
```

Comment:
right, that's why I was thinking python, which has a nice XML parser, which Rintze uses for the re-indent/reorder script anyway. I agree a regex would probably not be feasible, at least not reasonably so.